src/TimeToLive.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ResellerClub;
+
+use InvalidArgumentException;
+
+class TimeToLive
```

Comment:
There doesn't appear to a be toString method in this class. 
```
/**
     * Get the string representation of the TTL.
     *
     * @return string
     */
    public function __toString()
    {
        return $this->value;
    }
```

Reply:
Sure.

Why is it needed / when is it used?

Reply:
You'll need it to cast the value to a string or you would end up with an empty string, for example;
```
<?php

class foo
{
    private $value;

    public function __construct($value)
    {
        $this->value = $value;
    }
}
```

The calling the class; 
```
$test = new foo('wibble');
dd($test, (string) $test);
```

Yields; 
```
foo {#38 ▼
  +value: "wibble"
}
""
```